index.js

```diff
@@ -215,7 +215,7 @@ const sleepingTime = sleepingTimeInMinutes * 60000;
         try {
             console.log('START ', process.env.ACCOUNT, new Date().toLocaleString())
             const browser = await puppeteer.launch({
-                headless: true,
+                headless: process.env.HEADLESS || true,
```

Comment:
Thanks for contributing. 
I like the idea, but did you test it?
I think that when the variable is set to false, this condition will become true.
Also, I think the variables from the process.env are all read as string so in this case it won't be a boolean as expected by puppeteer.

I suggest creating another variable for the logic. Check how I did here for the quest:
https://github.com/alfficcadenti/splinterlands-bot/blob/bdc88d188edcff654fe1ba339fbc34f52a4b4c79/possibleTeams.js#L224

Reply:
Yup, I tested it, and I updated it

Reply:
👍 resolve the conflicts and I can merge it